content/CORE/CORETutorials/UsingTaskManager.md

```diff
@@ -1,7 +1,11 @@
 ---
 title: "Task Manager"
+description: "Use the Task Manager screen to display a list of tasks performed by your TrueNAS"
```

Comment:
Suggest this as an alternative description, also don't forget to use the proper emphasis for UI elements. 
```suggestion
description: "Use the **Task Manager** screen to display a list of tasks performed by your TrueNAS and to view logs related to system tasks."
```

Reply:
"Use the Task Manager screen to display a list of tasks performed by your TrueNAS and to view logs related to system tasks." All changes except bold because description is enclosed by quotes and will be interpreted literally.